rocAL/rocAL/include/pipeline/ring_buffer.h

```diff
@@ -47,14 +47,19 @@ class RingBuffer
     ///\param sub_buffer_count
     void init(RocalMemType mem_type, void *dev, std::vector<size_t> &sub_buffer_size);
     void initBoxEncoderMetaData(RocalMemType mem_type, size_t encoded_bbox_size, size_t encoded_labels_size);
+    void init_metadata(RocalMemType mem_type, std::vector<size_t> sub_buffer_size, unsigned sub_buffer_count);
```

Comment:
Is meta_data_sub_buffer_count required?

Reply:
@fiona-gladwin can you please explain here?

Reply:
Done

Reply:
Removed